instrumentation/akka-http-10.0/javaagent/akka-http-10.0-javaagent.gradle

```diff
@@ -51,16 +45,9 @@ dependencies {
   library "com.typesafe.akka:akka-http_2.11:10.0.0"
   library "com.typesafe.akka:akka-stream_2.11:2.4.14"
 
-  // There are some internal API changes in 10.1 that we would like to test separately for
-  version101TestImplementation "com.typesafe.akka:akka-http_2.11:10.1.0"
-  version101TestImplementation "com.typesafe.akka:akka-stream_2.11:2.5.11"
-}
-
-test.dependsOn version101Test
-
-compileVersion101TestGroovy {
```

Comment:
I couldn't find any difference between this pattern and the standard latestTestDeps, we generally are fine with it vs testing the baseline of a new minor I think.

Reply:
Ah to clarify, even if I didn't have to change to `testLibrary` for the mysterious 10.0.x failure, I would have still made this change

Reply:
Was able to fix issue by discarding entity